cairo_fuzz/diff-fuzz/src/main.rs

```diff
@@ -0,0 +1,109 @@
+#[macro_use]
+extern crate honggfuzz;
+use cairo_vm::{
+    cairo_run::{CairoRunConfig, cairo_run, write_encoded_trace},
+    hint_processor::builtin_hint_processor::builtin_hint_processor_definition::BuiltinHintProcessor,
+    vm::errors::{cairo_run_errors::CairoRunError, trace_errors::TraceError}
+};
+
+use std::{fs::{read, remove_file, File}, io::{Write, BufWriter}};
+use bincode::enc::write::Writer;
+
+fn main() {
+    loop {
+        fuzz!(|data: &[u8]| {
+            let trace_name_root = rand::random::<u64>().to_string();
+            let py_trace_name = trace_name_root.clone() + ".py.trace";
+            let rs_trace_name = trace_name_root + ".rs.trace";
+            let program_name = rand::random::<u64>().to_string();
+
+            match (execute_program(&rs_trace_name, &data), cairo_python(&program_name, &py_trace_name, &data)) {
+                (Ok(_), Ok(_)) => {
```

Comment:
why just check the cases when both return okey and when excecute_program fails with certain error and cairo python doesn´t?  this should panic in all cases when the result isn´t the same for both.

Reply:
Normally it should, however, there is a difference in how the rust VM and the python VM handle hints. For rust they are hardcoded and to replicate the python behaviour a different crate is needed ([cairo-rs-py](https://github.com/lambdaclass/cairo-rs-py)). Because of this, the fuzzer ignores the "Unknown hint" errors that might come from the rust implementation.